yaptide/persistence/models.py

```diff
@@ -356,9 +356,11 @@ class PageModel(db.Model):
 
     __tablename__ = 'Page'
     id: Column[int] = db.Column(db.Integer, primary_key=True)
+    page_name: Column[str] = db.Column(db.String, nullable=False, doc="Page name")
```

Comment:
keep in mind that this would reqiure flask-db-migrate stuff to be added as well.
I hope the users will still be able to load from DB their simulations saved prior to merging this PR - will they ? Will you maintain backward compatibility ?

Reply:
Currently it only works for new simulation. I will try to add backwards compatibility

Reply:
Backward compatibility will be crucial to merge this PR and roll it out to the production instance.